src/sysfiles.c

```diff
@@ -1019,16 +1023,20 @@ Int syFid;
 
 void syAnswerCont ( int signr )
 {
+#if !defined(LIBGAP)
     syStartraw( syFid );
     signal( SIGCONT, SIG_DFL );
     kill( getpid(), SIGCONT );
+#endif
 }
 
 void syAnswerTstp ( int signr )
 {
+#if !defined(LIBGAP)
```

Comment:
While you are here, perhaps mark both `syAnswerCont` and `syAnswerTstp` static?

Also, the two #ifdef's here should not be necessary, as `syAnswerCont` is only referenced in one place, namely `syAnswerTstp`; and that in turn is only reference once, too, in code that is already disabled for LIBGAP.